### PR TITLE
scoffolding code challenge

### PR DESCRIPTION
**Pull request by SianCulligan**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [x] Picture of whiteboard

### CODE REVIEW

File: code_challenges/401/breadthFirst/_tests_/breadth-first.test.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  });
+
+  it('An edge can be successfully added to the graph', () => {
+    let graph = new Graph();
+
+    graph.addNode('A');
+    graph.addNode('B');
+    graph.addEdge('A', 'B', 2);
+
+    expect(graph.nodes[0]).toMatchObject({
```
Comment (lizkavalski) [inline code_challenges/401/breadthFirst/_tests_/breadth-first.test.js:33]: change `nodes` to `adjList` because in your breath-frist.js your referring the nodes to adjList.

File: code_challenges/401/breadthFirst/_tests_/breadth-first.test.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+const Graph = require('../breadth-first');
+
+describe('Testing fnctionality of Graph', () => {
+  it('Node can be successfully added to the graph & can give me all the nodes', () => {
+    let graph = new Graph();
+    graph.addNode('A');
+    graph.addNode('B');
+    graph.addNode('C');
+    graph.addNode('D');
+    let nodes = graph.getNodes();
```
Comment (lizkavalski) [inline code_challenges/401/breadthFirst/_tests_/breadth-first.test.js:19]: change `.getNodes` to `. getadjList` because in your breadth-first.js that what your calling to get all the nodes.

File: code_challenges/401/breadthFirst/_tests_/breadth-first.test.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    expect(graph.size()).toBe(4);
+  })
+
+  it('A graph with only one node and edge can be properly returned', () => {
+    let graph = new Graph();
+
+    graph.addNode('A');
+    graph.addEdge('A', 'A', 5);
+
+    expect(graph.nodes[0]).toMatchObject({
```
Comment (lizkavalski) [inline code_challenges/401/breadthFirst/_tests_/breadth-first.test.js:80]: change `nodes` to `adjList` because in your breath-frist.js your referring the nodes to adjList.

File: code_challenges/401/breadthFirst/_tests_/breadth-first.test.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      val: 'A',
+      connections: [{ destVal: 'A', weight: 5 }],
+    });
+  });
+
+  it('An empty graph properly returns null', () => {
+    let graph = new Graph();
+    expect(graph.nodes).not.toBeDefined();
+  })
```
Comment (lizkavalski) [inline code_challenges/401/breadthFirst/_tests_/breadth-first.test.js:89]: missing  a test for breadthFirst function.
